importer/helper.go

```diff
@@ -90,7 +90,26 @@ func (h *ImportHelper) Import(db idb.IndexerDb, args []string) {
 		h.Log.Infof("%d blocks in %s, %.0f/s, %d txn, %.0f/s", blocks, dt.String(), float64(time.Second)*float64(blocks)/float64(dt), txCount, float64(time.Second)*float64(txCount)/float64(dt))
 	}
 
-	accountingRounds, txnCount := updateAccounting(db, h.DefaultFrozenCache, 0, h.GenesisJSONPath, h.NumRoundsLimit, h.Log)
+	state, err := db.GetImportState()
+	var startRound uint64 = 0
+	accountingRounds := 0
+	if err == idb.ErrorNotInitialized {
+		if InitialImport(db, h.GenesisJSONPath, h.Log) {
+			accountingRounds ++
```

Comment:
I think this shouldn't count as a round. Then `InitialImport()` doesn't need a return value and should probably be renamed to `MaybeImportGenesis()`.

Reply:
The initialize code is pretty strange. Something like this suggestion would be great for a next step. At the moment this function is used by the importer as well as the daemon, so it isn't quite so easy to change. Since the importer is deprecated removing it would probably be next, and then we can redo the init process.

Reply:
The daemon doesn't use the result of `InitialImport()`. The change should be easy.